fossilize_db.cpp

```diff
@@ -394,6 +404,8 @@ struct ZipDatabase : DatabaseInterface
 
 			if (!mz_zip_end(&mz))
 				LOGE("mz_zip_end failed!\n");
+
+			//TODO Use fadvise here?
```

Comment:
The replayer will only use StreamArchive. I don't think we need to change anything in the other implementations. They are mostly there for debugging purposes.